### ISSUE TITLE
Widescreen usability issue with button placement in file reload banner

### ISSUE THREAD

**Issue by cas--** [geany/geany#706]
I am not sure what you specifically call the reload banner but here is a screenshot, pointing to the issue:

![core py - -home-calum-projects-autoshutdown-autoshutdown - autoshutdown plugin - geany_036](https://cloud.githubusercontent.com/assets/606038/10635467/dfec3608-77f0-11e5-8b7b-b5797d699103.png)

Overall I really  happy that this banner was introduced and it was something I was hoping would be implemented. The problem I am finding is that at fullscreen the buttons are placed far too far away from any other part of the application and require significant extra mouse movement. 

It might be that gtk constrains what is possible without writing a custom widget but what I would propose is that if possible place the buttons on the lefthand side. 

I made a quick mockup on my proposal, also reducing the amount of vertical space the banner takes up:
![core py - -home-calum-projects-autoshutdown-autoshutdown - autoshutdown plugin - geany_0362](https://cloud.githubusercontent.com/assets/606038/10636591/a6098e60-77f8-11e5-903e-07605a242bf7.png)

**Comment by elextr** [comment]
You are right that layout of the infobar is controlled by GTK.  You could ask them for the alternate style.  

I doubt that we would want to maintain our own custom widget in both GTK2 and GTK3 even if "somebody" contributed one.

**Comment by cas--** [comment]
Ah infobar, of course and looking at it in gtk docs I do see it is a very basic widget with few options. That is a shame but I totally understand not wanting to have custom widgets to maintain.

What about rewording the dialog in the meantime? When doing that mockup I realised that the would help from being a little clearer for the user.

**Comment by elextr** [comment]
I think the current message is just the one that existed before infobars were added and I'm not sure the old dialog had all three options, but anyway making the options more explicit is good.

Maybe "Reload the buffer from disk or overwrite the disk with the buffer, or neither?"?

`<start bikeshed>` ;-P

**Comment by codebrainz** [comment]
@cas-- it's not exactly a solution if you like the mouse (like me), but there are some fast keybindings for it. Like for the "externally modified" info bar, `Esc` is like "cancel", `Ctrl+R` is "reload" (works with info bar), and `Ctrl+S` ought to be "save" (possible bug, doesn't work with info bar).

**Comment by b4n** [comment]
(probably not a solution you'd like either, but on your screen about half the width is wasted anyway, so you could shrink the Geany window to a useful size, have the buttons close to the rest of the UI, and as a bonus display some other useful thing on the right :])

**Comment by kugel-** [comment]
You can also press tab to cycle through the insofar buttons (activite with space) and the text.

**Comment by kugel-** [comment]
s/insofar/info bar/
